common.vars.pkr.hcl

```diff
@@ -1 +1 @@
-postgres-version = "15.1.1.94"
+postgres-version = "15.1.1.95"
```

Comment:
Can you check `ansible/vars.yaml` not sure if it's on this branch.

Reply:
Could you check again? It's showing up on my end now

<img width="1115" alt="image" src="https://github.com/user-attachments/assets/b0507534-30f8-4da9-8a60-ad286895c2c3">